src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -32,6 +32,17 @@
  * order of the elements or add new elements on the end.
  * </p>
  * <p>
+ * The check allows leaving out the comma at the end if either the left
+ * or right curly braces are on the same line.
```

Comment:
This explanation is weird as it miss critical part: ".....same line as trailing comma"
please update other places too.